http-server/src/main/java/org/triplea/server/access/ApiKeyGenerator.java

```diff
@@ -39,7 +41,7 @@ private static String randomHash() {
   }
 
   @Override
-  public ApiKey get() {
+  public ApiKey apply(final InetAddress inetAddress) {
```

Comment:
Why is the parameter added here?
Doesn't seem to be used anywhere.

Reply:
Nope, it's not. This class will be deleted entirely in short order. The wiring is 'future looking'. It's basically the same as a feature flag, any code behind a feature flag is just as dead as this parameter.

Reply:
The class is marked with:
```
// TODO: Project#12 This is copy/pasted from lobby sub-project
```
I'm skipping adding a new TODO here as the whole class will be deleted in future iterations.

The other location that was of question, I added a TODO to mark that we will use the unused parameter in future iterations.